diff --git a/node_modules/react-native-spinkit-fix-new/android/build.gradle b/node_modules/react-native-spinkit-fix-new/android/build.gradle
index 7bbf59e..f6acdf1 100644
--- a/node_modules/react-native-spinkit-fix-new/android/build.gradle
+++ b/node_modules/react-native-spinkit-fix-new/android/build.gradle
@@ -1,6 +1,7 @@
 apply plugin: 'com.android.library'
 
 repositories {
+    google()
     mavenCentral()
     maven {
         url "https://jitpack.io"
@@ -9,30 +10,32 @@ repositories {
 
 buildscript {
     repositories {
+        google()
         mavenCentral()
         maven {
             url "https://jitpack.io"
         }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.0.0'
+        classpath 'com.android.tools.build:gradle:8.1.0'
     }
 }
 
 android {
-    compileSdkVersion 30
-    buildToolsVersion "28.0.3"
+    namespace "com.react.rnspinkit"
+    compileSdkVersion 35
+    buildToolsVersion "35.0.0"
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 28
+        minSdkVersion 24
+        targetSdkVersion 35
         versionCode 1
         versionName "1.0"
     }
     buildTypes {
         release {
             minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
 }
@@ -40,5 +43,5 @@ android {
 
 dependencies {
     implementation 'com.facebook.react:react-native:+'
-    implementation 'com.github.ybq:Android-SpinKit:1.2.0'
+    implementation 'com.github.ybq:Android-SpinKit:1.4.0'
 }
diff --git a/node_modules/react-native-spinkit-fix-new/android/src/main/AndroidManifest.xml b/node_modules/react-native-spinkit-fix-new/android/src/main/AndroidManifest.xml
index 5ff6d07..ea92a28 100644
--- a/node_modules/react-native-spinkit-fix-new/android/src/main/AndroidManifest.xml
+++ b/node_modules/react-native-spinkit-fix-new/android/src/main/AndroidManifest.xml
@@ -1,5 +1,4 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.react.rnspinkit">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
     <application
         android:label="@string/app_name"
