diff --git a/node_modules/react-native-scrollable-tab-view-fix-guo/DefaultTabBar.js b/node_modules/react-native-scrollable-tab-view-fix-guo/DefaultTabBar.js
index 4cb92de..5320bc2 100644
--- a/node_modules/react-native-scrollable-tab-view-fix-guo/DefaultTabBar.js
+++ b/node_modules/react-native-scrollable-tab-view-fix-guo/DefaultTabBar.js
@@ -1,5 +1,6 @@
 const React = require('react');
-const { ViewPropTypes } = ReactNative = require('react-native');
+const ReactNative = require('react-native');
+const { ViewPropTypes } = require('deprecated-react-native-prop-types');
 const PropTypes = require('prop-types');
 const createReactClass = require('create-react-class');
 const {
diff --git a/node_modules/react-native-scrollable-tab-view-fix-guo/ScrollableTabBar.js b/node_modules/react-native-scrollable-tab-view-fix-guo/ScrollableTabBar.js
index 0a49160..102b9c7 100644
--- a/node_modules/react-native-scrollable-tab-view-fix-guo/ScrollableTabBar.js
+++ b/node_modules/react-native-scrollable-tab-view-fix-guo/ScrollableTabBar.js
@@ -1,5 +1,6 @@
 const React = require('react');
-const { ViewPropTypes } = ReactNative = require('react-native');
+const ReactNative = require('react-native');
+const { ViewPropTypes } = require('deprecated-react-native-prop-types');
 const PropTypes = require('prop-types');
 const createReactClass = require('create-react-class');
 const {
diff --git a/node_modules/react-native-scrollable-tab-view-fix-guo/index.js b/node_modules/react-native-scrollable-tab-view-fix-guo/index.js
index 437e4b3..9d0e0dd 100644
--- a/node_modules/react-native-scrollable-tab-view-fix-guo/index.js
+++ b/node_modules/react-native-scrollable-tab-view-fix-guo/index.js
@@ -1,6 +1,7 @@
 const React = require('react');
 const { Component } = React;
-const { ViewPropTypes } = ReactNative = require('react-native');
+const ReactNative = require('react-native');
+const { ViewPropTypes } = require('deprecated-react-native-prop-types');
 const createReactClass = require('create-react-class');
 const PropTypes = require('prop-types');
 const {
