diff --git a/node_modules/react-native-version-number-fix-new/android/build.gradle b/node_modules/react-native-version-number-fix-new/android/build.gradle
index 2e55ebb..3f8a2c1 100644
--- a/node_modules/react-native-version-number-fix-new/android/build.gradle
+++ b/node_modules/react-native-version-number-fix-new/android/build.gradle
@@ -2,7 +2,7 @@
 apply plugin: 'com.android.library'
 
 android {
-    namespace "com.reactnativeversioncheck"
+    namespace "com.apsl.versionnumber"
     compileSdkVersion 35
     buildToolsVersion "35.0.0"
 
