diff --git a/node_modules/react-native-scrollable-tab-view-fix-guo/DefaultTabBar.js b/node_modules/react-native-scrollable-tab-view-fix-guo/DefaultTabBar.js
index 4cb92de..9abc4f4 100644
--- a/node_modules/react-native-scrollable-tab-view-fix-guo/DefaultTabBar.js
+++ b/node_modules/react-native-scrollable-tab-view-fix-guo/DefaultTabBar.js
@@ -1,5 +1,6 @@
 const React = require('react');
-const { ViewPropTypes } = ReactNative = require('react-native');
+const ReactNative = require('react-native');
+const { ViewPropTypes, TextPropTypes } = require('deprecated-react-native-prop-types');
 const PropTypes = require('prop-types');
 const createReactClass = require('create-react-class');
 const {
@@ -18,7 +19,7 @@ const DefaultTabBar = createReactClass({
     backgroundColor: PropTypes.string,
     activeTextColor: PropTypes.string,
     inactiveTextColor: PropTypes.string,
-    textStyle: Text.propTypes.style,
+    textStyle: TextPropTypes.style,
     tabStyle: ViewPropTypes.style,
     renderTab: PropTypes.func,
     underlineStyle: ViewPropTypes.style,
diff --git a/node_modules/react-native-scrollable-tab-view-fix-guo/ScrollableTabBar.js b/node_modules/react-native-scrollable-tab-view-fix-guo/ScrollableTabBar.js
index 0a49160..88714b5 100644
--- a/node_modules/react-native-scrollable-tab-view-fix-guo/ScrollableTabBar.js
+++ b/node_modules/react-native-scrollable-tab-view-fix-guo/ScrollableTabBar.js
@@ -1,5 +1,6 @@
 const React = require('react');
-const { ViewPropTypes } = ReactNative = require('react-native');
+const ReactNative = require('react-native');
+const { ViewPropTypes, TextPropTypes } = require('deprecated-react-native-prop-types');
 const PropTypes = require('prop-types');
 const createReactClass = require('create-react-class');
 const {
@@ -27,7 +28,7 @@ const ScrollableTabBar = createReactClass({
     style: ViewPropTypes.style,
     tabStyle: ViewPropTypes.style,
     tabsContainerStyle: ViewPropTypes.style,
-    textStyle: Text.propTypes.style,
+    textStyle: TextPropTypes.style,
     renderTab: PropTypes.func,
     underlineStyle: ViewPropTypes.style,
     onScroll: PropTypes.func,
diff --git a/node_modules/react-native-scrollable-tab-view-fix-guo/index.js b/node_modules/react-native-scrollable-tab-view-fix-guo/index.js
index 437e4b3..9d0e0dd 100644
--- a/node_modules/react-native-scrollable-tab-view-fix-guo/index.js
+++ b/node_modules/react-native-scrollable-tab-view-fix-guo/index.js
@@ -1,6 +1,7 @@
 const React = require('react');
 const { Component } = React;
-const { ViewPropTypes } = ReactNative = require('react-native');
+const ReactNative = require('react-native');
+const { ViewPropTypes } = require('deprecated-react-native-prop-types');
 const createReactClass = require('create-react-class');
 const PropTypes = require('prop-types');
 const {
